src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button !== 0) {
+      return;
+    }
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject(new Error());
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.addEventListener('contextmenu', (eventt) => {
```

Comment:
```suggestion
  document.addEventListener('contextmenu', (event) => {
```

Reply:
> Не коректно трохи працює, другий проміс має резолвитись, якщо або ліва або права кнопка мишки була натиснута а в тебе тільки коли права

still not working left click for second promise(